src/ImmersedBoundaries/conditional_fluxes.jl

```diff
@@ -44,25 +97,25 @@ const ATD = AbstractTimeDiscretization
 
 # dx(uu), dy(vu), dz(wu)
 # ccc,    ffc,    fcf
-@inline _advective_momentum_flux_Uu(i, j, k, ibg::GFIBG, args...) = conditional_flux_ccc(i, j, k, ibg, advective_momentum_flux_Uu, args...)
-@inline _advective_momentum_flux_Vu(i, j, k, ibg::GFIBG, args...) = conditional_flux_ffc(i, j, k, ibg, advective_momentum_flux_Vu, args...)
-@inline _advective_momentum_flux_Wu(i, j, k, ibg::GFIBG, args...) = conditional_flux_fcf(i, j, k, ibg, advective_momentum_flux_Wu, args...)
+@inline _advective_momentum_flux_Uu(i, j, k, ibg::GFIBG, closure, U, args...) = conditional_flux_ccc(i, j, k, ibg, advective_momentum_flux_Uu, closure, U, args...)
+@inline _advective_momentum_flux_Vu(i, j, k, ibg::GFIBG, closure, U, args...) = conditional_flux_ffc(i, j, k, ibg, advective_momentum_flux_Vu, closure, U, args...)
+@inline _advective_momentum_flux_Wu(i, j, k, ibg::GFIBG, closure, U, args...) = conditional_flux_fcf(i, j, k, ibg, advective_momentum_flux_Wu, closure, U, args...)
 
 # dx(uv), dy(vv), dz(wv)
 # ffc,    ccc,    cff
-@inline _advective_momentum_flux_Uv(i, j, k, ibg::GFIBG, args...) = conditional_flux_ffc(i, j, k, ibg, advective_momentum_flux_Uv, args...)
-@inline _advective_momentum_flux_Vv(i, j, k, ibg::GFIBG, args...) = conditional_flux_ccc(i, j, k, ibg, advective_momentum_flux_Vv, args...)
-@inline _advective_momentum_flux_Wv(i, j, k, ibg::GFIBG, args...) = conditional_flux_cff(i, j, k, ibg, advective_momentum_flux_Wv, args...)
+@inline _advective_momentum_flux_Uv(i, j, k, ibg::GFIBG, closure, disc::ATD, clock, U, args...) = conditional_flux_ffc(i, j, k, ibg, advective_momentum_flux_Uv, closure, disc::ATD, clock, U, args...)
+@inline _advective_momentum_flux_Vv(i, j, k, ibg::GFIBG, closure, disc::ATD, clock, U, args...) = conditional_flux_ccc(i, j, k, ibg, advective_momentum_flux_Vv, closure, disc::ATD, clock, U, args...)
```

Comment:
Do you need `closure` and `disc` in the advective fluxes?

Reply:
You're right, I don't. Just removed it